packages/elements-core/src/components/Docs/Model/Model.tsx

```diff
@@ -45,7 +45,9 @@ const ModelComponent: React.FC<ModelProps> = ({ data: unresolvedData, className,
 
   const description = (
     <>
-      {data.description && <MarkdownViewer className="sl-mb-6" markdown={data.description} />}
+      {data.description && data.type === 'object' && (
+        <MarkdownViewer className="sl-mb-6" role="textbox" markdown={data.description} />
```

Comment:
I'm personally not a fan of specifying a `role` since there are ARIA-roles embedded in DOM elements by default, although I understand the need of accesibility here since `MarkdownViewer` is a `div`.

Reply:
General thought, not limited to this PR - as a part of tech debt, maybe we should remove this hardcoded roles and implement semantic HTML a bit more

Reply:
Is it okay for me to leave this as is for now? I'm happy to update it, just couldn't think of another way to grab only this part.

Reply:
Let's leave this here for now, it doesn't affect the PR itself directly